resume-buddy/src/main/java/com/google/sps/ServletHelpers.java

```diff
@@ -8,8 +8,8 @@
   private ServletHelpers() {}
 
   /**
-   * @return the request parameter, or the default value if the parameter was not specified by
-   * the client
+   * @return the request parameter, or the default value if the parameter was not specified by the
+   *     client
```

Comment:
Hm, how did this happen? Auto-formatter?

Reply:
yeah something always happens to this line with the autoformatter :( but I think it's fixed now